package.json

```diff
@@ -111,7 +111,7 @@
   "scripts": {
     "tsc": "tsc",
     "tslint": "tslint --project tsconfig.json ./*.ts ./lib/**/*.ts ./test/**/*.ts ./types/**/*.ts",
-    "test": "npm run tsc && npm run tslint && nyc mocha ./dist/test/**/*.js -t 10000 --reporter mocha-junit-reporter --reporter spec",
+    "test": "npm run tsc && npm run tslint && jest --ci --reporters=default --reporters=jest-junit",
```

Comment:
Does Jest have code coverage? Does it use `NYC` so we can see coverage for `.ts` files?

Reply:
it has, we use in the portal

Reply:
yeah it looks exactly the same as how it was being output before